### PR TITLE
make drag&drop todos from/to contexts work again  - fix #2054

### LINKED ISSUE #2054: drag & drop between contexts  broken
tried drag & drop functionality between contexts on version 2.3.0 & on master-branch.
From looking at the code it seems that this has been working at some point in time, but isn't now.
Seems that the "drop_target"'s are missing

### PR DESCRIPTION
**Pull request by scriptythekid**
fix #2054

### CODE REVIEW

**Comment by hirocaster** [comment]
LGTM

File: app/controllers/todos_controller.rb [hunk 1]
```diff
@@ -397,7 +397,7 @@ def change_context
     @todo = current_user.todos.find(params[:id])
     @original_item_context_id = @todo.context_id
     @original_item = current_user.todos.build(@todo.attributes)  # create a (unsaved) copy of the original todo
-    @context = current_user.contexts.find(params[:todo][:context_id])
+    @context = current_user.contexts.find(params[:todo][:context_id].sub! 'c', '')
```
Comment (mattr-) [inline app/controllers/todos_controller.rb]: why would there be a 'c' in the context id from the form?

Comment (scriptythekid) [reply app/controllers/todos_controller.rb]: no clue, they were present in my vanilla install, and were preventing drag & drop from working.

**Comment by ZeiP** [comment]
The 'c' prefix was caused by the way id was specified in _context.html.erb. Refactored the context elements to use the same underscore format that todos use instead to make it less hacky.

**Comment by ZeiP** [comment]
Finally merged the change. Thanks, @scriptythekid!

**Pull request merged**